image-index.md

```diff
@@ -63,7 +63,18 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`variant`** *string*
 
-        This OPTIONAL property specifies the variant of the CPU, for example `armv6l` to specify a particular CPU variant of the ARM CPU.
+        This OPTIONAL property specifies the variant of the CPU.
+        Image indexes SHOULD use, and implementations SHOULD understand, values listed in the following table.
+        When the variant of the CPU is not listed in the table, values are implementation-defined and SHOULD be submitted to this specification for standardization.
+
+        | ISA/ABI                              | `architecture` | `variant`   |
+        |--------------------------------------|----------------|-------------|
+        | ARM 32-bit, v5 and earlier           | Unsupported    | Unsupported |
+        | ARM 32-bit, v6 (non-Raspberry Pi 1)  | Unsupported    | Unsupported |
```

Comment:
I agree - but I would replace 'unsupported' with 'out of scope' to make it explicit that this spec does not cover older ARM ISAs.